### ISSUE TITLE
Create .npmrc in dist/electron/UnPackaged even if the project doesn't have one

### ISSUE THREAD

**Issue by yusufkandemir** [quasarframework/quasar#17504]
We check if `.npmrc` exists before copying it here:
https://github.com/quasarframework/quasar/blob/c310cb8e4ffa74d3d0ae6ad023032142518aa7e7/app-vite/lib/modes/electron/electron-builder.js#L86-L104

If it doesn't exist, for example, when `.npmrc` is in the root folder in a monorepo, it won't be copied. So, pnpm would use symlinks and the build would fail on Windows, for example.
It won't be respected in monorepos without passing `--ignore-workspace` to unpackaged install params, but it's needed for that scenario, it's complex.

**Solution**
So, we should create a `.npmrc` file with the relevant options in `dist/electron/UnPackaged` even if the project doesn't have a `.npmrc` file.

**Workaround**
Create an empty `.npmrc` file in the project folder.

**Comment by rstoenescu** [comment]
Will be available in q/app-vite 2.0.0-beta.22 & q/app-webpack 4.0.0-beta.23

**Comment by yusufkandemir** [comment]
I was going to handle this one, but thanks 👍 I was going to update the matching to:
```
# ❌ don't match this, it's just mentioning the option in a comment:
# I am just mentioning node-linker here

# ❌ don't match this, it's commented out:
# node-linker=some-value

# ✅ match this
node-linker=hoisted
```
But, we even have to override the value as the user might need to use a specific node-linker option in their project. If we don't override it, the user will have a bad time as they will have to give up on something on their project, or it's just not going to work.
